coderd/coderd.go

```diff
@@ -606,7 +659,7 @@ func New(options *Options) *API {
 		ResumeTokenProvider:     api.Options.CoordinatorResumeTokenProvider,
 	})
 	if err != nil {
-		api.Logger.Fatal(api.ctx, "failed to initialize tailnet client service", slog.Error(err))
+		must(api.Logger, "failed to initialize tailnet client service", err)
```

Comment:
you're checking for a nil error which `must` also does, so I don't think `must` adds much clarity if you use it like this.